package-lock.json

```diff
@@ -236,9 +264,9 @@
       "dev": true
     },
     "acorn": {
-      "version": "7.1.0",
-      "resolved": "https://registry.npmjs.org/acorn/-/acorn-7.1.0.tgz",
-      "integrity": "sha512-kL5CuoXA/dgxlBbVrflsflzQ3PAas7RYZB52NOm/6839iVYJgKMJ3cQJD+t2i5+qFa8h3MDpEOJiS64E8JLnSQ==",
+      "version": "7.1.1",
```

Comment:
This was the dependency that had an issue. 🙄 
Never heard of it. Don't know what it does from the _useless_ name. 
https://www.npmjs.com/package/acorn
A JavaScript Parser written in JS, apparently. 🤷‍♂